modules/navigation/2d/nav_mesh_generator_2d.cpp

```diff
@@ -117,8 +117,8 @@ void NavMeshGenerator2D::sync() {
 }
 
 void NavMeshGenerator2D::cleanup() {
-	baking_navmesh_mutex.lock();
-	generator_task_mutex.lock();
+	MutexLock baking_navmesh_lock(baking_navmesh_mutex);
+	MutexLock generator_task_lock(generator_task_mutex);
```

Comment:
Despite these will be unlocked in reverse order, matching the former code, I'd still put the second lock in a scope, for clarity.